docs/rackhd/configuration.rst

```diff
@@ -166,6 +167,8 @@ The following table describes the configuration parameters in config.json:
       - Address for RackHD WebSocket Service to bind to (defaults to '0.0.0.0').
     * - wssBindPort
       - Listening port for RackHD WebSocket Service (defaults to 9100).
+    * - trustedProxy
+      - Enable trust proxy in express. Populate req.ip with most left in XforwardFor list
```

Comment:
Worth adding a link to ExpressJS documentation here? https://expressjs.com/en/guide/behind-proxies.html